compiler/src/diagnostics/comment_attributes.re

```diff
@@ -1,11 +1,15 @@
-exception InvalidAttribute(string);
-exception MalformedAttribute(string, string);
+open Grain_parsing;
 
 type param_id =
-  | LabeledParam(string)
-  | PositionalParam(int);
+  | LabeledParam(string, Location.t)
```

Comment:
These locs looks unused, unless I missed it?

Reply:
Not used. I was mostly setting up the opportunity to warn/error specifically on the name/position.